h2o-r/h2o-package/R/import.R

```diff
@@ -194,9 +194,12 @@ h2o.uploadFile <- function(path, destination_frame = "",
 #' @param username Username for SQL server
 #' @param password Password for SQL server
 #' @param columns (Optional) Character vector of column names to import from SQL table. Default is to import all columns. 
-#' @param optimize (Optional) Optimize import of SQL table for faster imports. Experimental. Default is true. 
+#' @param optimize (Optional) Optimize import of SQL table for faster imports. Experimental. Default is true.
+#' @param streaming (Optional) Turn on to disable distributed import. Streaming mode reads the data sequentially
+#'        using a single H2O node and then streams it to the rest of the nodes of the cluster.
+#'        Can be used for databases that do not support OFFSET-like clauses in SQL statements.
 #' @export
-h2o.import_sql_table <- function(connection_url, table, username, password, columns = NULL, optimize = NULL) {
+h2o.import_sql_table <- function(connection_url, table, username, password, columns = NULL, optimize = NULL, streaming = NULL) {
```

Comment:
don't forget to deprecate optimize here as well, and replace streaming flag by new param